src/pages/showcase/index.tsx

```diff
@@ -78,11 +44,8 @@ function ShowcaseCards() {
 	return (
 		<section className="margin-top--lg margin-bottom--xl">
 			<div className="container">
-				<div className={clsx("margin-bottom--md")}>
-					<Heading as="h2">Features ✨</Heading>
-				</div>
 				<ul className={clsx("container", "clean-list", styles.cards)}>
-					{features.map(feature => ShowcaseCard(feature))}
+					{plugins.map(feature => ShowcaseCard(feature))}
```

Comment:
It's better to keep the same name: `{plugins.map(plugin => ShowcaseCard(plugin))}`

Reply:
But I think these are different categories... argh fine. I'll do that when I consolidate these into components.

Reply:
What should I call a component for this? FeaturesDisplay?

Reply:
Hmm seems we only need a plugin section, without the need to differentiate between user plugins or features. This way we just need to keep the current structure.

Reply:
Sorry I don't understand.